aws_lambda_powertools/utilities/parser/envelopes/__init__.py

```diff
@@ -1,7 +1,15 @@
 from .base import BaseEnvelope
 from .dynamodb import DynamoDBStreamEnvelope
 from .event_bridge import EventBridgeEnvelope
+from .kinesis import KinesisEnvelope
 from .sns import SnsEnvelope
 from .sqs import SqsEnvelope
 
-__all__ = ["DynamoDBStreamEnvelope", "EventBridgeEnvelope", "SnsEnvelope", "SqsEnvelope", "BaseEnvelope"]
+__all__ = [
+    "DynamoDBStreamEnvelope",
+    "EventBridgeEnvelope",
+    "KinesisEnvelope",
```

Comment:
```suggestion
    "KinesisDataStreamEnvelope",
```

Because we might want to support Kinesis Firehose, Kinesis Analytics, etc.